apps/ltn/src/draw_cells.rs

```diff
@@ -245,7 +247,12 @@ impl RenderCellsBuilder {
             // can just clip the result.
             let mut clipped = Vec::new();
             for p in cell_polygons {
-                clipped.extend(p.intersection(&result.boundary_polygon));
+                // If clipping fails, just use the original polygon.
+                if let Ok(list) = p.intersection(&result.boundary_polygon) {
+                    clipped.extend(list);
+                } else {
```

Comment:
It's definitely up to your discretion, but do you want to log these kinds of failures?

I can imagine it being tricky to track down strange geometry bugs if they're now being silently ignored, especially if they were previously crashing. If you're worried about log spam, even just a `debug!`  (or `trace!`) message which normal users wouldn't see might be helpful if you're actively trying to understand strange behavior.

Reply:
In general, maybe. In this case, there will be a very loud visual bug anyway. Logs could be helpful if I ask people to include them in bug reports, but in general, it's easier to just reproduce what they're seeing